zulipterminal/cli/run.py

```diff
@@ -219,9 +208,26 @@ def main(options: Optional[List[str]]=None) -> None:
                            format(", ".join(complete))))
         print("   autohide setting '{}' specified {}."
               .format(*zterm['autohide']))
+        # For binary settings
+        # Specify setting in order True, False
+        valid_settings = {
+            'autohide': ['autohide', 'no_autohide'],
+        }
+        settings = dict()
```

Comment:
Perhaps type-annotate this, which will indicate the expected types with less need for a comment?

Along similar lines, you could rename this to `boolean_settings`?